app/components/post_draft/post_draft.tsx

```diff
@@ -2,13 +2,10 @@
 // See LICENSE.txt for license information.
 
 import React, {useEffect, useState} from 'react';
-import {Platform} from 'react-native';
 
 import Autocomplete from '@components/autocomplete';
-import {ExtraKeyboard} from '@context/extra_keyboard';
```

Comment:
I see this component is no longer being used, but we did not delete the component from the codebase?